.scripts/linux/install-benchmarks.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+set -x
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+REPO_DIR=$SCRIPT_DIR/../../
+cd $REPO_DIR
+
+# Check out the library.
+git clone https://github.com/google/benchmark.git
```

Comment:
Should we incorporate this step into the docker image we use?

Reply:
Depends if we view the docker image as a CI tool or a vw distribution mechanism